tests/test_integrated_channels/test_integrated_channel/test_transmitters/test_course_metadata.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the base course metadata transmitter.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import unittest
+
+import mock
+from integrated_channels.integrated_channel.transmitters.course_metadata import CourseTransmitter
+from pytest import mark
+
+from test_utils import factories
+
+
+@mark.django_db
+class TestCourseTransmitter(unittest.TestCase):
+    """
+    Tests for the class ``CourseTransmitter``.
+    """
+
+    def setUp(self):
+        super(TestCourseTransmitter, self).setUp()
+        enterprise_customer = factories.EnterpriseCustomerFactory(name='Starfleet Academy')
+        self.enterprise_config = factories.SAPSuccessFactorsEnterpriseCustomerConfigurationFactory(
+            enterprise_customer=enterprise_customer,
+            key="client_id",
+            sapsf_base_url="http://test.successfactors.com/",
+            sapsf_company_id="company_id",
+            sapsf_user_id="user_id",
+            secret="client_secret",
+        )
+        self.global_config = factories.SAPSuccessFactorsGlobalConfigurationFactory()
```

Comment:
@UmanShahzad This code isn't part of the `tests/test_integrated_channels/test_sap_success_factors/` package; should it be referencing SAP-specific classes?

Reply:
We need some non-abstract configuration for these things to work, so it's okay for it be to any arbitrary channel (atm between SAPSF/Degreed).

Reply:
@UmanShahzad OK, could this be mentioned in a docstring (at class or method level)?

Reply:
Yep! :+1: